source/simulator/core.cc

```diff
@@ -2023,6 +2025,8 @@ namespace aspect
     CitationInfo::print_info_block (pcout);
 
     stokes_matrix_free.reset();
+
+    pcout << "   Total wallclock time elapsed including restarts:" << round(walltime.wall_time()+walltime_from_resume) << "s" << std::endl;
```

Comment:
```suggestion
    pcout << "-- Total wallclock time elapsed including restarts:" << round(walltime.wall_time()+walltime_from_resume) << "s" << std::endl;
```

Reply:
I see, thanks

Reply:
You need to do the same in the other places as well.